addons/sale/controllers/portal.py

```diff
@@ -115,6 +115,8 @@ def portal_order_page(
         message=False,
         download=False,
         downpayment=None,
+        link_amount=None,
+        installment=None,
```

Comment:
Why do not you reuse the downpayment param instead of 'installment' or use 'installment' everywhere? I can see that they serve the same job or I miss smth?

Reply:
They don't. A downpayment will always result in confirming the quotation, and the installment might be lower than the confirmation amount. I think it is more clear to use separate variables.

Reply:
But even if it is a downpayment but the link is generated then you still use `installment`  , no?

Reply:
Indeed, after some looking at it with fresher eye, indeed it could be merged together, which results in heavy simplification of this PR.